system-tests/common.py

```diff
@@ -316,28 +309,27 @@ def tear_down(self):
             shutil.rmtree(self._repo_path)
 
 
-class CheckTask(systest.TestCase, metaclass=ABCMeta):
+class CheckTask(object, metaclass=ABCMeta):
```

Comment:
I don't think it's necessary to explicitly inherit from `object` but it does no harm.

There is now a convenience class `ABC` you can inherit from. Check the documentation.